recipes/blocked-matrix-utils/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "blocked-matrix-utils" %}
+{% set version = "1.0.4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 550f5f7ee147b513abca70432cd85b0fe4fc87218e0447e9d538db33ec00d02d
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+requirements:
+  host:
+    - numpy
```

Comment:
Is `numpy` required to build the package? It doesn't look like it from the `setup.py`

Reply:
So the host section is about build dependency? Sorry, I am a novice here and confused. No, this is pure Python, so should I remove it?

Reply:
See: https://conda-forge.org/docs/maintainer/adding_pkgs.html#requirements